rllab/misc/tensor_summary.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+import numpy as np
+import tensorflow as tf
+from tensorboard import summary as summary_lib
+from tensorboard.plugins.custom_scalar import layout_pb2
+
+from rllab.misc.console import mkdir_p
+
+
+class Summary:
```

Comment:
I think `TensorBoardOutput` or `TensorBoardOutputFormat` would be a better name for this class.